core-networking/build.gradle.kts

```diff
@@ -1,35 +1,44 @@
+import com.codingfeline.buildkonfig.compiler.FieldSpec
```

Comment:
Adding `BuildKonfig` to deal with the build config variables in multiplatform world